plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/status/StatusPlexusResource.java

```diff
@@ -74,7 +74,7 @@ public String getResourceUri() {
 
   @Override
   public PathProtectionDescriptor getResourceProtection() {
-    return new PathProtectionDescriptor(getResourceUri(), "authcNxBasic,perms[nexus:status]");
+    return new PathProtectionDescriptor(getResourceUri(), "authcBasic,perms[nexus:status]");
```

Comment:
authcNxBasic is the only perm that causes sessions to be created. login resource and status resource were the only ones that had it. Now with this change, only the login resource creates sessions.